docs/website/docs/intro.md

```diff
@@ -62,10 +69,11 @@ external APIs, backends or containers, scales on micro and large infra alike.
 - User-friendly, declarative interface that removes knowledge obstacles for beginners
 while empowering senior professionals.
 
-## Getting started with `dlt`
+## Getting hands-on with `dlt`
+
 1. Play with the
 [Google Colab demo](https://colab.research.google.com/drive/1NfSB1DpwbbHX9_t5vlalBTf13utwpMGx?usp=sharing).
-This is the simplest way to see `dlt` in action.
+This is the simplest way to see `dlt` in action. You don't need GCP, just a regualr Google account.
```

Comment:
> You don't need GCP, just a regualr Google account.

I think this info is not really necessary, users can figure out that quickly themselves, also it's quite obvious